src/lib/api/conversions.ts

```diff
@@ -0,0 +1,54 @@
+import { RemoteItem } from '@dcl/builder-client'
+import { Rarity } from '@dcl/schemas'
+import { Item, ItemRarity } from 'modules/item/types'
+
+export function fromRemoteItem(remoteItem: RemoteItem) {
+  const item: Item = {
+    id: remoteItem.id,
+    name: remoteItem.name,
+    thumbnail: remoteItem.thumbnail,
+    owner: remoteItem.eth_address,
+    description: remoteItem.description ?? '',
+    isPublished: remoteItem.is_published,
+    isApproved: remoteItem.is_approved,
+    inCatalyst: remoteItem.in_catalyst,
+    type: remoteItem.type,
+    data: remoteItem.data,
+    contents: remoteItem.contents,
+    contentHash: remoteItem.content_hash,
+    metrics: remoteItem.metrics,
+    createdAt: +new Date(remoteItem.created_at),
+    updatedAt: +new Date(remoteItem.created_at)
+  }
+
+  if (remoteItem.collection_id) item.collectionId = remoteItem.collection_id
+  if (remoteItem.blockchain_item_id) item.tokenId = remoteItem.blockchain_item_id
+  if (remoteItem.price) item.price = remoteItem.price
+  if (remoteItem.urn) item.urn = remoteItem.urn
+  if (remoteItem.beneficiary) item.beneficiary = remoteItem.beneficiary
+  if (remoteItem.rarity) item.rarity = rarityToItemRarity(remoteItem.rarity)
+  if (remoteItem.total_supply !== null) item.totalSupply = remoteItem.total_supply // 0 is false
+
+  return item
+}
+
+function rarityToItemRarity(rarity: Rarity): ItemRarity {
```

Comment:
What if we add a verb or perposition at the beginning of this method to show intent? `fromRarityToItemRarity`, `(convert/transform)RarityToItemRarity`?